pages/writing/writing.md

```diff
@@ -131,6 +131,9 @@ using Pkg
 Pkg.rm("Example")
 ```
 
+The blog post
+[Pkg.jl and Julia Environments for Beginners](https://jkrumbiegel.com/pages/2022-08-26-pkg-introduction/) will get you comfortable with the details.
+
 ### Shell mode (`;`)
 
 By pressing `;` you enter a terminal, where you can execute any bash command you want.
```

Comment:
Is it always a bash shell, even on Mac where the default is zsh and Windows where the default is PowerShell?

Reply:
I said bash but I don't know shit about the subtleties of terminals ^^ any help is welcome

Reply:
Me either 🤷🏻‍♂️. I'm pretty sure Mac ditched Bash though. Maybe better to say "shell command" for accuracy?